src/ForkCMS/Component/Module/CopyModulesToOtherLocaleResults.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace ForkCMS\Component\Module;
+
+final class CopyModulesToOtherLocaleResults
+{
+    /**
+     * @var array
+     */
+    private $idMap;
+
+    /**
+     * @var array
+     */
+    private $extraIdMap;
+
+    public function add(string $moduleName, array $idMap, array $extraIdMap): void
+    {
+        $this->idMap[$moduleName] = $idMap;
+        $this->extraIdMap[$moduleName] = $extraIdMap;
+    }
+
+    public function getExtraId(string $moduleName, $id)
+    {
+        return $this->getValue($this->extraIdMap, $moduleName, $id);
+    }
+
+    public function getExtraIds(string $moduleName)
+    {
+        return $this->getValues($this->extraIdMap, $moduleName);
+    }
+
+    public function getId(string $moduleName, $id)
+    {
+        return $this->getValue($this->idMap, $moduleName, $id);
+    }
+
+    public function getIds(string $moduleName)
+    {
+        return $this->getValues($this->idMap, $moduleName);
+    }
+
+    private function getValue(array $map, string $moduleName, $id)
+    {
+        if (!array_key_exists($moduleName, $map)) {
+            throw new \Exception(
```

Comment:
use statement for `Exception`
or better yet: write a custom exception

Reply:
Custom Exception added